classes/person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permession
+
+  def initialize(age, name = 'unkown', parent_permession: true)
+    @id = rand
```

Comment:
- **[OPTIONAL]** This `rand` only returns random numbers between 0 and 1. It would be of better utility to return more meaningful random numbers, that are integers.

  ```
  @id = rand(1...1000)
  ```